### PR TITLE
Set up associations

### PR DESCRIPTION
**Pull request by herokudev**
- Create a class Classroom.
- Create a class Book.
- Create a class Rental.
- Set up associations between classes and objects.

### CODE REVIEW

File: classroom.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  attr_accessor :label
+  attr_reader :students
+
+  def add_student(student)
+    return if @students.include?(student)
+
+    @students.push(student)
+    student.classroom = self
+  end
+end
```
Comment (safafa) [inline classroom.rb:16]: image

- [ ] As mentioned in the project requirements. You should create both sides of the has-many/belongs-to relationship between Classroom and Student. well done on creating the has-many side. Kindly Create the belongs-to side (a student belongs to a classroom) in the Student class.
- [ ] [OPTIONAL] I think the code might look better if you place the attr_accessor and reader at the top of the class

File: rental.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    @date = date
+    @person = person
+    person.rentals << self
+    @book = book
+    book.rentals << self
+  end
+
+  attr_accessor :date
+  attr_reader :person, :book
+end
```
Comment (safafa) [inline rental.rb:12]: image

- [ ] As mentioned in the project's requirements you should Create the has-many side of Person and Rental (a person has many rentals). For the moment, you only have the belongs-to side of Rental and Person (a rental belongs to a person). Kindly modify the person class too

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @herokudev , You did a great job so far 👏🏼 👏🏼 

- Linter checks passed.
- Most of associations are implemented.

However, here are some points that you need to work on to get your project approved.

## Required changes :

- [ ] Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mke2111 — APPROVED** [review]
### Hi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**